Formula/disco.rb

```diff
@@ -11,12 +11,26 @@ class Disco < Formula
     sha256 "003628e6ba324264409fb54c457089ead2d1c2490565e6a52fd01f19f4ffbaed" => :mountain_lion
   end
 
+  option "without-test", "Skip build-time tests (not recommended)"
+
   depends_on :python if MacOS.version <= :snow_leopard
   depends_on "erlang"
   depends_on "simplejson" => :python if MacOS.version <= :leopard
   depends_on "libcmph"
 
-  # Modifies config for single-node operation
+  # .travis.yml changed after the tag but before the Erlang 18.1 patch
+  patch do
+    url "https://github.com/discoproject/disco/commit/14e726f1627ad3e7f1767722f36fb966e244185b.patch"
+    sha256 "94b23cd201f10b141bb839e4169a4648d4d6a10d95dccee243aad15558e1ae41"
+  end
+
+  # Upstream PR adding Erlang 18.1 (passed their Travis Erlang 18.1 CI)
+  patch do
+    url "https://github.com/discoproject/disco/pull/632.patch"
+    sha256 "17f52a7e942b3913b2be2b362d29fd308dc253af211fa82cae38cddb0678f525"
+  end
+
+  # Modifies config for single-node operation.
   patch :DATA
```

Comment:
@mikemcquaid I'm not sure it's intent but it seems to only be about configuration. I believe @apjanke added the original patch, so maybe he can shed some light.